src/MCPServer/lib/utils.py

```diff
@@ -0,0 +1,18 @@
+
+import logging
+
+LOGGER = logging.getLogger('archivematica.mcp.server')
+
+def log_exceptions(fn):
+    """
+    Decorator to wrap a function in a try-catch that logs the exception.
+
+    Useful for catching exceptions in threads, which do not normally report back to the parent thread.
+    """
+    def wrapped(*args,  **kwargs):
```

Comment:
Do we want to use `functool.wraps` to give the wrapper the original function's name? Not sure what this will do to tracebacks.